.github/workflows/deploy.yml

```
@@ -46,20 +46,43 @@ jobs:
       with:
         node-version: '18'
 
-    - name: Run Database Migration - Upload Puzzles
+    - name: Set up Python for DVC
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)